### PR TITLE
mdls: add page

### PR DESCRIPTION
**Pull request by liamrosenfeld**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/mdls.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# mdls
+
+> Lists the metadata attributes for a file.
+
+- Get all metadata for file:
```
Comment (navarroaxel) [inline pages/osx/mdls.md:5]:
```suggestion
- Display the list of metadata attributes for file:
```

File: pages/osx/mdls.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`mdls {{file}}`
```
Comment (navarroaxel) [inline pages/osx/mdls.md:7]:
```suggestion
`mdls {{path/to/file}}`
```

File: pages/osx/mdls.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Get specific metadata attribute:
```
Comment (navarroaxel) [inline pages/osx/mdls.md:9]:
```suggestion
- Display a specific metadata attribute:
```

File: pages/osx/mdls.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+`mdls -name {{attribute}} {{file}}`
```
Comment (navarroaxel) [inline pages/osx/mdls.md:11]:
```suggestion
`mdls -name {{attribute}} {{path/to/file}}`
```

File: pages/osx/mdls.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# mdls
+
+> Lists the metadata attributes for a file.
```
Comment (navarroaxel) [inline pages/osx/mdls.md:3]:
```suggestion
> Display the metadata attributes for a file.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @liamrosenfeld, welcome and thanks for your submission. I left some comments.

**Comment by liamrosenfeld** [comment]
Thank you for all those improvements! I squashed them into my commit.

**Review by navarroaxel — APPROVED** [review]
Thanks @liamrosenfeld, please don't `push --force` the next time. it's harder track the suggestions.

**Review by sbrl — APPROVED** [review]
Thanks, @liamrosenfeld! Looks ok to me. Welcome to tldr pages :-)

**Pull request merged**